### PR TITLE
[SDK] Airdrops

### PR DESCRIPTION
**Pull request by kumaryash90**

### CODE REVIEW

File: packages/sdk/src/evm/core/classes/airdrop-erc1155.ts [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        args: [tokenAddress, tokenOwner, contents],
+        parse: (receipt) => {
+          const events = this.contractWrapper.parseLogs<AirdropFailedEvent>(
+            "AirdropFailed",
+            receipt.logs,
+          );
+          return events.map((e) => {
+            const recipient = e.args.recipient;
+            return {
+              recipient,
```
Comment (joaquim-verges) [inline packages/sdk/src/evm/core/classes/airdrop-erc1155.ts:54]: how do you expect ppl to use this return value here?

File: packages/sdk/src/evm/core/classes/airdrop-erc1155.ts [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+   */
+  getAddress(): Address {
+    return this.contractWrapper.readContract.address;
+  }
+
+  /** ******************************
+   * WRITE FUNCTIONS
+   *******************************/
+
+  drop = /* @__PURE__ */ buildTransactionFunction(
```
Comment (joaquim-verges) [inline packages/sdk/src/evm/core/classes/airdrop-erc1155.ts:40]: add docs with `@twfeature AirdropERC1155`

File: packages/sdk/src/evm/core/classes/airdrop-erc1155.ts (hunk 1 above)
Comment (kumaryash90) [reply packages/sdk/src/evm/core/classes/airdrop-erc1155.ts:54]: As a list of recipients where airdrop failed.

Comment (kumaryash90) [reply packages/sdk/src/evm/core/classes/airdrop-erc1155.ts]: Renamed it

File: packages/sdk/src/evm/core/classes/airdrop-erc721.ts [hunk 3]
```diff
@@ -38,7 +33,11 @@ export class Airdrop721<T extends IAirdropERC721 | AirdropERC721>
    *******************************/
 
   drop = /* @__PURE__ */ buildTransactionFunction(
-    async (tokenAddress: string, tokenOwner: string, contents: Airdrop721Content[]): Promise<Transaction<{recipient: string}[]>> => {
+    async (
+      tokenAddress: string,
+      tokenOwner: string,
+      contents: Airdrop721Content[],
+    ): Promise<Transaction<{ failedRecipient: string; tokenId: number }[]>> => {
```
Comment (joaquim-verges) [inline packages/sdk/src/evm/core/classes/airdrop-erc721.ts:74]: do a type for this, and should also have the success ones
{
successfulDrops: { ... }[]
failedDrops: { ... }[]
}

**Pull request merged**